src/main/java/org/opensearch/flowframework/transport/handler/WorkflowFunction.java

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+package org.opensearch.flowframework.transport.handler;
+
+/**
+ * Interface for a workflow function.
+ *
+ */
+public interface WorkflowFunction {
```

Comment:
What does this interface give us that we don't already have with [`Runnable`](https://docs.oracle.com/javase/8/docs/api/java/lang/Runnable.html)?